hw/dv/tools/ralgen/ralgen.py

```diff
@@ -62,8 +62,9 @@ def main():
         if hjson_path:
             args += ["--hjson-path", root_dir / hjson_path]
         if alias_hjson:
+            args += ["--alias-files"]
             for alias in alias_hjson:
-                args += ["--alias-files", root_dir / alias]
+                args += [root_dir / alias]
```

Comment:
nit: list comp: `args += ["--alias-files"] + [root_dir / alias for alias in alias_hjson]`